deeplearning4j/deeplearning4j-core/src/test/java/org/deeplearning4j/gradientcheck/SDLossCustomFunctions/SDLossMAE.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2020 Skymind, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+package org.deeplearning4j.gradientcheck.SDLossCustomFunctions;
```

Comment:
Missed this in the review: java packages should be lowercase
so `org.deeplearning4j.gradientcheck.sdlosscustom` or something